synapse/lib/stormlib/gen.py

```diff
@@ -97,6 +97,12 @@ class LibGen(s_stormtypes.Lib):
                       {'name': 'name', 'type': 'str', 'desc': 'The name of the language.'},
                   ),
                   'returns': {'type': 'storm:node', 'desc': 'A lang:language node with the given name.'}}},
+        {'name': 'langByCode', 'desc': 'Returns a lang:language node by language code, adding the node if it does not exist.',
+         'type': {'type': 'function', '_funcname': '_storm_query',
+                  'args': (
+                      {'name': 'name', 'type': 'str', 'desc': 'The language code for the language.'},
```

Comment:
Do we want to add a `try` arg to allow normalization to fail?

Reply:
Sure, i'll add that 👍